docs/zh/cli-client/gov/submit-proposal.md

```diff
@@ -46,7 +46,7 @@ iriscli gov submit-proposal --chain-id=test --title="update MinDeposit proposal"
 ### 提交一个'SoftwareUpgrade'类型的提议
 
 ```shell
-iriscli gov submit-proposal --chain-id=test --title="irishub0.7.0 upgrade proposal" --type=SoftwareUpgrade --description="a new software upgrade proposal" --from=node0 --fee=0.01iris --software=https://github.com/irisnet/irishub/tree/v0.9.0 --version=2 --switch-height=80 --threshold=0.9
+iriscli gov submit-proposal --chain-id=test --title="irishub0.7.0 upgrade proposal" --type=SoftwareUpgrade --description="a new software upgrade proposal" --from=node0 --fee=0.01iris --software=https://github.com/irisnet/irishub/tree/v0.9.0 --version=2 --switch-height=80
```

Comment:
SoftwareUpgrade proposal must have `threshold` flag. Please add it.

Reply:
fixed

Reply:
SoftwareUpgrade proposal must have threshold flag.